satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -162,4 +162,18 @@ read limitoffset (
 )
 
 update overlay_cache_node ( where overlay_cache_node.key = ? )
-delete overlay_cache_node ( where overlay_cache_node.key = ? )
\ No newline at end of file
+delete overlay_cache_node ( where overlay_cache_node.key = ? )
+model injuredsegment (
+	key info
```

Comment:
I don't think we need a primary key on this table.

Reply:
I think it returns an error with no key. From the dbx readme under *Models*, "key is required and declares the primary key for the model"

Reply:
ok, nevermind then